.github/labeler.yml

```diff
@@ -3,12 +3,13 @@
   - plugins/postcss-base-plugin/**
   - rollup/**
 
-"packages/postcss-plugins-values-parser":
-  - packages/postcss-plugins-values-parser/**
-
 "plugin-packs/postcss-preset-env":
   - plugin-packs/postcss-preset-env/**
 
+"cli":
+  - cli/csstools-cli/**
+  - packages/base-cli/**
```

Comment:
simply "cli" is fine I think, but maybe you have a better suggestion? :)

I also added "packages/base-cli/" as it will always impact the cli.

Reply:
Thanks!